strings/revenant_names.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
run this through [jsonlint ](https://jsonlint.com/)please

Reply:
yes please lint this